EZSwiftExtensionsTests/IntTests.swift

```diff
@@ -79,4 +86,26 @@ class IntTests: XCTestCase {
         XCTAssertEqual(value1.toInt32, expected1)
         XCTAssertEqual(value2.toInt32, expected2)
     }
+    
+    func testString() {
+        
+        let value1: Int = 399
+        let expected1: String = "399"
+        let value2: Int = -30
+        let expected2: String = "-30"
+        let value3: Int = 0
+        let expected3: String = "0"
+        
+        XCTAssertEqual(value1.toString, expected1)
+        XCTAssertEqual(value2.toString, expected2)
+        XCTAssertEqual(value3.toString, expected3)
+    }
+    
+    func testRange() {
+        
+        let value: Int = 0
+        let expected: CountableRange<Int> = 0..<value
```

Comment:
do not rely on value, create expected range with constants.

Also we need some at least test where range is valid :)